ray-operator/apis/ray/v1/rayjob_types.go

```diff
@@ -114,7 +115,6 @@ type RayJobStatus struct {
 	RayClusterStatus RayClusterStatus `json:"rayClusterStatus,omitempty"`
 	// observedGeneration is the most recent generation observed for this RayJob. It corresponds to the
 	// RayJob's generation, which is updated on mutation by the API Server.
-	// +optional
```

Comment:
What's the reason for this change?

Reply:
I just found that it is useless. There is no change in the CRD YAML after I remove the `+optional`.